src/main/java/org/candlepin/subscriptions/tally/InventoryAccountUsageCollector.java

```diff
@@ -86,8 +87,15 @@ public InventoryAccountUsageCollector(FactNormalizer factNormalizer,
         Map<String, Host> hypervisorHosts = new HashMap<>();
         Map<String, Integer> hypervisorGuestCounts = new HashMap<>();
 
-        inventory.reportedHypervisors(accounts,
-            reported -> hypMapping.put((String) reported[0], (String) reported[1]));
+        Consumer<Object[]> addToHypervisorMap = resultRow -> {
+            final int hypIdIndex = 0;
+            final int hypSubmanIdIndex = 1;
```

Comment:
When we make that glossary, we should include the different kind of ids that exist in the swatch hosts table